### PR TITLE
54 API schema file

### PR DESCRIPTION
**Pull request by Ninolina88**
API schema file created

### CODE REVIEW

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
Index file is fine. Most of the rest of them is incorrectly mapped. Please refer to TMDB documentation for correct mappings.
Also, did you find the postman collection?

**Comment by Ninolina88** [comment]
> Index file is fine. Most of the rest of them is incorrectly mapped. Please refer to TMDB documentation for correct mappings. Also, did you find the postman collection?

Yes I did find the postman collection. I will correct the mappings.

File: public/media-hunt/API/discover.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const ROOT = "/discover";
+
+export default {
+  get: {
+    discover: {
+      url: "/discover/movie",
+    },
```
Comment (pvukmanovic-pannonians) [inline public/media-hunt/API/discover.js:7]: Add support for both movie and TV. Add movie and tv under discover, and have them both have their own url.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
If you won't be using ROOT constants remove them

File: public/media-hunt/API/discover.js (hunk 1 above)
Comment (Ninolina88) [reply public/media-hunt/API/discover.js]: i put root to use, and adjusted discover as per request. Hopefully its fine this time :) .

File: public/media-hunt/API/discover.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const ROOT = {
+  get: {
+    discover: {
+      url: "/discover/movie",
+      url: "/discover/tv"
```
Comment (pvukmanovic-pannonians) [inline public/media-hunt/API/discover.js:5]: Object can't have two keys with same name. VsCode has probably highlighted that as well with red underline. Please name them movie and tv appropriately.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]

File: public/media-hunt/API/discover.js (hunk 2 above)
Comment (Ninolina88) [reply public/media-hunt/API/discover.js:5]: got it, will be adjusted with the next commit.

File: public/media-hunt/API/discover.js (hunk 1 above)
Comment (Ninolina88) [reply public/media-hunt/API/discover.js]: root removed

File: public/media-hunt/API/discover.js (hunk 2 above)
Comment (Ninolina88) [reply public/media-hunt/API/discover.js]: added genres, top rated and reviews to movie and tv, and discover.js file removed, and placed in movie and tv js files.
Changes are commited.

Comment (Ninolina88) [reply public/media-hunt/API/discover.js]: i noticed that from API/index.js discover file needs to be removed. I will remove it.

Comment (Ninolina88) [reply public/media-hunt/API/discover.js]: > 

discover file is removed from index.js

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]

File: public/media-hunt/API/tv.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export default {
```
[the remark this answers is not in the document]
Comment (Ninolina88) [reply public/media-hunt/API/tv.js:1]: joined, and the code formatted properly. Also, tv season is removed from the index.

**Review by pvukmanovic-pannonians — APPROVED** [review]
Lgtm

**Pull request merged**